src/collectors/windows.plugin/metadata.yaml

```
@@ -0,0 +1,265 @@
+plugin_name: windows.plugin
+modules:
+  - meta:
+      plugin_name: windows.plugin
+      module_name: PerflibProcesses
+      monitored_instance:
+        name: System statistics
+        link: "https://www.microsoft.com/en-us/windows"
+        categories:
+          - data-collection.windows-systems
+        icon_filename: "windows.svg"
+      related_resources:
+        integrations:
+          list: []
+      info_provided_to_referring_integrations:
+        description: ""
+      keywords:
+        - process counts
+        - threads
+        - context switch
+      most_popular: false
+    overview:
+      data_collection:
+          metrics_description: |
+            Perflib provides different statistical methods about Microsoft Windows environment. This collector query for
+            Object 'System' to show actual number of processes, threads and context switches.
+          method_description: ""
+      supported_platforms:
+        include: ["windows"]
+        exclude: []
+      multi_instance: false
+      additional_permissions:
+        description: ""
+      default_behavior:
+          auto_detection:
+            description: |
+              The collector auto-detects all metrics. No configuration is needed.
+          limits:
+            description: ""
+          performance_impact:
+            description: ""
+    setup:
+      prerequisites:
+        list: []
+      configuration:
+          file:
+            section_name: "plugin:windows"
+            name: "netdata.conf"
+            description: ""
+          options:
+            description: ""
+            folding:
+              title: ""
+              enabled: true
+            list: []
+          examples:
+            folding:
+              enabled: true
+              title: ""
+            list: []
+    troubleshooting:
+      problems:
+        list: []
+    alerts: []
+    metrics:
+      folding:
+        title: Metrics
+        enabled: false
+      description: ""
+      availability: []
+      scopes:
+        - name: global
+          description: ""
```

scopes need to have a description, check https://github.com/netdata/netdata/blob/282836b8ef99ac51eabe1a3b13c5945eb6b43968/src/collectors/python.d.plugin/openldap/metadata.yaml#L173

```suggestion
          description: "These metrics refer to the entire monitored application."
```